Julia/Manifest.toml

```diff
@@ -0,0 +1,249 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Is this file created automatically?  Is it something that should go into the repository?

Reply:
Yes, this file is created automatically, but it is standard to include it in Julia repositories in order to track the dependencies of the packages one uses in their project. These files together enable correct code loading via `Pkg.instantiate` . Here is some [further reading](https://pkgdocs.julialang.org/v1/toml-files/) if you are curious.

Reply:
Thank you for the explanation and link.  Very helpful.